target/arc/semfunc.c

```diff
@@ -55,27 +55,52 @@ arc_gen_cmpl2_i64(TCGv_i64 ret, TCGv_i64 arg1,
 }
 
 void
+/**
```

Comment:
`/**` -> `/*`

https://qemu.readthedocs.io/en/latest/devel/style.html#comment-style

AND put it beofore the `void` line.